docs/tutorials/01-simple-viewer/04-viewer.mdx

```diff
@@ -18,8 +18,6 @@ Let's start by implementing the Forge Viewer functionality for our application.
 Create a `viewer.js` file under the `wwwroot` subfolder with the following code:
 
 ```js title="wwwroot/viewer.js"
-/// import * as Autodesk from "@types/forge-viewer";
```

Comment:
What's the reason for removing the directive? We include it in order to provide static typing for the viewer classes (https://twitter.com/ipetrbroz/status/1327240538697961472). I do agree though that we should probably clarify this in the tutorial (incl. the fact that it only works for Node.js projects when you install the `@types/forge-viewer` dependency. How about adding this to a `:::tip` note?

Reply:
Hey Petr, shoot, my bad! I thought that line was left in there erroneously and didn't realize we had Typescript support. Will fix that and add a note!